wdl-format/src/token/post.rs

```diff
@@ -29,8 +34,14 @@ pub enum PostToken {
     /// One indentation.
     Indent,
 
+    /// A temporary indent.
+    ///
+    /// This is added after a [`PostToken::Indent`] during the formatting of
+    /// command sections.
```

Comment:
It's not clear to me from these comments why this is needed. Maybe there is additional documentation below where the command block is formatted, but it may be useful to expand here.